Core/Base/AgentModule.py

```diff
@@ -22,6 +23,8 @@
 from DIRAC.FrameworkSystem.Client.MonitoringClient import gMonitor
 from DIRAC.ConfigurationSystem.Client import PathFinder
 from DIRAC.FrameworkSystem.Client.MonitoringClient import MonitoringClient
+from DIRAC.Core.Utilities.ThreadScheduler import gThreadScheduler
```

Comment:
Actually we never used a gThreadScheduler in an Agent as far as I know. @fstagni @atsareg any reason why this would not work ?